### DISCUSSION TITLE
Keyboard shortcut for duplicating the current tab

### DISCUSSION THREAD

**Discussion by tothambrus11** [zen-browser/desktop#3748] [Ideas]
It would be great if it was possible to duplicate the current tab with a shortcut, like Ctrl+Shift+K in Edge. Also, another shortcut cut duplicate the current tab into a split view, so that we can start comparing different sections of the same page easily. In VSCode, this functionality is used a lot for code editing (split left, split right, split up, split down), and I find that the same happens often during browsing, e.g. while editing a large document in google docs.

**Comment by kvenn** [comment]
I use this feature all the time in Arc and Vivaldi. I have mine bound to ctrl+D. Couple example use cases:
- Generating charts in Amplitude / GA and can quickly duplicate the current chart and make modifications
- Same concept for amazon searches or notion docs

It lets you keep your place on a website, while creating another tab you can easily make modifications to.

I know you can press the middle mouse button (the wheel) to duplicate, but a keyboard shortcut would be much better (as I'm on a trackpad)

**Reply by jeregon1** [reply]
I'm on the same page os you guys, love the shortcut!

**Comment by denispol** [comment]
It's an awesome feature, especially for those of us who use tab containers!

**Comment by szulim** [comment]
I would also add a use case in which I want to duplicate tab to another space

**Comment by kniziol** [comment]
It would be fantastic to have that keyboard shortcut. It’s a much-needed and missing feature. As we can see, there are numerous discussions about the "duplicate tab" feature: https://github.com/zen-browser/desktop/discussions?discussions_q=is%3Aopen+%22duplicate+tab%22.

Cheers ✌️

**Comment by mr-cheffy** [comment]
Done with https://github.com/zen-browser/desktop/pull/13123
